<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadDTA</class>
 <widget class="QDialog" name="DownloadDTA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download DTA</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Adresse der DTA-Datei:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="editURL">
        <property name="toolTip">
         <string>aktuelle Daten der Wärmepumpe: http://&lt;ip-addresse&gt;/proclog</string>
        </property>
        <property name="text">
         <string>http://192.168.2.20/proclog</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Download Verzeichnis:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editDir"/>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="btnDir">
        <property name="toolTip">
         <string>Verzeichnis auswählen</string>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QCheckBox" name="cbRename">
        <property name="text">
         <string>Datei umbenennen? (Format 'JJMMTT_HHMM_proclog.dta')</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Status:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblStatus">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Eingabe</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editURL</tabstop>
  <tabstop>editDir</tabstop>
  <tabstop>btnDir</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DownloadDTA</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>410</x>
     <y>191</y>
    </hint>
    <hint type="destinationlabel">
     <x>435</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
